runtime/src/impls.rs

```diff
@@ -455,7 +455,7 @@ impl pallet_election_provider_multi_phase::Config for Runtime {
 	type Fallback = onchain::OnChainExecution<OnChainSeqPhragmen>;
 	type ForceOrigin = EnsureRoot<AccountId>;
 	type GovernanceFallback = onchain::OnChainExecution<OnChainSeqPhragmen>;
-	type MaxWinners = constants::MaxActiveValidators;
+	type MaxWinners = constants::staking::MaxActiveValidators;
```

Comment:
Could we remove `constants::MaxActiveValidators` if that is not used anymore?

Reply:
Its already removed